src/main/java/uk/gov/pay/ledger/report/params/TransactionSummaryParams.java

```diff
@@ -13,13 +13,13 @@
 import static org.apache.commons.lang3.StringUtils.isNotBlank;
 
 public class TransactionSummaryParams {
-    private static final String TO_DATE_FIELD = "to_date";
-    private static final String GATEWAY_ACCOUNT_EXTERNAL_FIELD = "account_id";
-    private static final String FROM_DATE_FIELD = "from_date";
 
     @QueryParam("account_id")
     private String accountId;
 
+    @QueryParam("moto")
```

Comment:
Maybe this name could be slightly more representative of what it does. Possibly along the lines of `include_moto_statistics`